docs/photutils/aperture.rst

```diff
@@ -160,9 +160,8 @@ them into one `~astropy.table.Table`::
 
     >>> from astropy.table import hstack
     >>> phot_table = hstack(flux)
-    >>> print(phot_table['aperture_sum_1', 'aperture_sum_2', 'aperture_sum_3'])    # doctest: +FLOAT_CMP
+    >>> print(phot_table['aperture_sum_1', 'aperture_sum_2', 'aperture_sum_3'])    # doctest: +SKIP
```

Comment:
Unfortunately I don't see any way to control the formatting for the output of a table.  @bsipocz Please let me know if you have any other ideas.